dali/test/dali_test_single_op.h

```diff
@@ -135,17 +135,21 @@ void StringToVector(const char *name, const char *val, OpSpec *spec, DALIDataTyp
 }
 
 
-// Define a virtual base class for single operator tests,
-// where we want to add a single operator to a pipeline,
-// run the pipe using known data, and compare the result to
-// a reference solution.
-//
-// Implementaions must define:
-//  OpSpec AddOp() - definition of the operator
-//  void SetInputs() - define all external inputs to the graph
-//  void GetOutputs() - retrieve all testable outputs from the graph
-//  bool Compare() - Compare against a (supplied) reference implementation
-template <typename ImgType>
+/**
+ * Virtual base class for single operator tests.
+ * 1. Adds single operator to pipeline
+ * 2. Runs the pipe using known data
+ *    (specified in class, that extends DALISingleOpTest)
+ * 3. Compares result to reference solution (also specified in subclass)
+ *
+ * Example usage is to overload Reference(...) function in subclass,
```

Comment:
I think that previous comment about what parts should be implemented was better that this example use case. Could you revert it, or fix the old comment if it was outdated?